server/src/main/java/com/cloud/alert/AlertManagerImpl.java

```diff
@@ -706,14 +707,13 @@ public void sendAlert(AlertType alertType, long dataCenterId, Long podId, Long c
             newAlert.setName(alertType.getName());
             _alertDao.persist(newAlert);
         } else {
-            if (s_logger.isDebugEnabled()) {
-                    s_logger.debug("Have already sent: " + alert.getSentCount() + " emails for alert type '" + alertType + "' -- skipping send email");
-                }
+            logger.debug("Have already sent: " + alert.getSentCount() + " emails for alert type '" + alertType + "' -- skipping send email");
```

Comment:
why remove the level check?
```suggestion
            if (logger.isDebugEnabled()) {
                logger.debug("Have already sent: " + alert.getSentCount() + " emails for alert type '" + alertType + "' -- skipping send email");
            }
```

Reply:
@DaanHoogland My intention is to reduce cyclomatic complexity, as this check is only executed because we did not update to Log4J yet. Also, for this situation, the computing costs is irrelevant.